### ISSUE TITLE
Add spectra to report function?

### ISSUE THREAD

**Issue by jona-sassenhagen** [mne-tools/mne-python#5237]
Maybe it would make sense to add spectra to the reports. They can be very informative.

Another idea would be epochs images, which can also be highly diagnostic.

Not volunteering :)

@jasmainak

**Comment by jasmainak** [comment]
I agreee ;)

**Comment by dengemann** [comment]
Yes @jona-sassenhagen @jasmainak this is a good idea. At least for the power spectra :)

**Comment by agramfort** [comment]
@jona-sassenhagen still not volunteering?

**Comment by jona-sassenhagen** [comment]
As you see, I just opened 3 other PRs instead 8)

**Comment by larsoner** [comment]
Given how easy it is to manually add `matplotlib` and `mayavi` figures using the `add_*` functions, I'm not sure we should add too many of these. PSD calculation on raw can take a while, too. But I agree this is almost always useful, so on the balance I'm +0.5 on adding it. I might have time to try

**Comment by agramfort** [comment]
yes it takes time but I typically compute it on the first minute of data
and not the full recording. we should do this by default to limit computing
cost.

**Comment by larsoner** [comment]
> yes it takes time but I typically compute it on the first minute of data
> and not the full recording. we should do this by default to limit computing
> cost.

I disagree. The `Report` function should follow the defaults of `raw.plot_psd`, otherwise it's unexpected behavior. The way I coded it, it's easy for you to change this behavior during rendering if you want. FWIW the default of `plot_psd` actually originally had a limited `tmax` but we decided to change this at some point to use the entire recording.
